tests/app/notifications/test_process_client_response.py

```diff
@@ -105,7 +105,16 @@ def test_process_sms_updates_sent_by_with_client_name_if_not_in_noti(notify_db,
     assert sample_notification.sent_by == 'mmg'
 
 
-def test_process_sms_does_not_update_sent_by_if_already_set(mocker, notify_db, sample_notification):
+def test_process_sms_updates_billable_units_if_zero(sample_notification):
+    sample_notification.billable_units = 0
```

Comment:
Should we use the create_notification() method here for pytest4 compatibility?

Reply:
This should be pytest 4 compatible - I think it's only incompatible if you call a fixture like a function by doing something like `sample_notification()`